src/main/play-doc/operation/Install.md

```diff
@@ -1113,46 +982,13 @@ Service ready.
 
 After updating the HAProxy configuration file, ConductR-HAProxy will signal HAProxy to reload for the updated configuration.
 
-#### RHEL and CentOS: disable requiretty
-
-On RHEL and CentOS it may also be neccessary to [disable default requiretty](https://bugzilla.redhat.com/show_bug.cgi?id=1020147) for the named execution user in `sudoers`.
-
-```bash
-[10.0.7.118]$ echo 'Defaults: conductr-agent  !requiretty' | sudo tee -a /etc/sudoers
-```
+ConductR-HAProxy the HAProxy reload script is located at `/usr/bin/reloadHAProxy.sh` by default. However on CoreOS the `/usr` is a read-only volume, and thus the HAProxy reload script need to be made available elsewhere.
```

Comment:
^ For, With or In or remove
For ConductR-HAProxy, the HAProxy reload script is...

and thus the HAProxy reload script 'needs to be' or 'must be made'